### PR TITLE
Match args in function calls and definitions

### PR DESCRIPTION
**Pull request by adibalaram**
I've matched the arguments in function calls and definitions for all files. Also called get_BQ to households.py since this was missing.

### CODE REVIEW

**Comment by adibalaram** [comment]
@jdebacker I think this is ready to be merged

**Comment by jdebacker** [comment]
@adibalaram Can you sync to the most recent changes to the master branch of this repo?  I don't think you are reflecting all the changes from two recently merged PRs.

**Comment by adibalaram** [comment]
I just synced. Is there anything else I need to do?

**Comment by jdebacker** [comment]
@adibalaram I don't know.  I'm waiting to see if the tests pass on Travis CI (which is what is displayed here).  

If you want to run tests locally, then from the `./scripts` folder in your terminal type `python -m pytest`.

**Comment by jdebacker** [comment]
@adibalaram I don't know.  I'm waiting to see if the tests pass on Travis CI (which is what is displayed here).  

If you want to run tests locally, then from the `./scripts` folder in your terminal type `python -m pytest`.

File: scripts/households.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def FOC_labor(c, n_s, args):
     Returns:
     foc_errors_n: A list of n2, n3, ..., nS
     '''
-    sigma, r, w, l_tilde, chi, b_ellipse, upsilon = args
+    sigma, w, l_tilde, chi, b_ellipse, upsilon = args
```
Comment (jdebacker) [inline scripts/households.py:77]: @adibalaram you are absolutely right that `r` doesn't need to be in here.  Removing it is causing a test to fail.  To make this test pass, you need to go to `tests/test_households.py` and remove `r` from `n_args` on line 17.

**Pull request closed without merging**